src/codegen.cpp

```diff
@@ -709,10 +709,11 @@ typedef struct {
 
 typedef struct {
     size_t len;
+    // variable length
     struct {
         int64_t isref;
         Function *f;
-    } data[];
+    } data[1];
```

Comment:
I'm pretty sure this change isn't ISO-conforming C. The old version should have been more correct.